pennylane_qiskit/qiskit_device.py

```diff
@@ -114,31 +94,44 @@ class QiskitDevice(Device, abc.ABC):
     _state_backends = {"statevector_simulator", "unitary_simulator"}
     """set[str]: Set of backend names that define the backends
     that support returning the underlying quantum statevector"""
+    _hw_backends = {"qasm_simulator", "ibmq_qasm_simulator"}
```

Comment:
I think the reason `_hw_backends` wasn't defined previously is that the available hardware backends are server dependent, and become available independently of Qiskit releases (e.g., `ibm_rome` became available just today). By hardcoding them in here, will that cause issues?

Reply:
Oh you're right! Thanks for this! :blush: 
```python
IBMQ.get_provider().backends()
```
Got this:
```python
[<IBMQSimulator('ibmq_qasm_simulator') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmqx2') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_16_melbourne') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_vigo') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_ourense') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_london') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_burlington') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_essex') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_armonk') from IBMQ(hub='ibm-q', group='open', project='main')>,
 <IBMQBackend('ibmq_rome') from IBMQ(hub='ibm-q', group='open', project='main')>]
```